CondTools/L1TriggerExt/python/L1SubsystemParamsExt_cfi.py

```diff
@@ -41,6 +41,12 @@ def initL1SubsystemsExt( tagBaseVec = [],
             type = cms.string('L1TMuonEndCapForest'),
             key = cms.string(objectKey)
         ),
+        cms.PSet(
+            record = cms.string('L1TMuonOverlapFwVersionO2ORcd'),
+            tag = cms.string('L1TMuonOverlapFwVersion_' + tagBaseVec[ L1CondEnumExt.L1TMuonOverlapFwVersion ]),
+            type = cms.string('L1TMuonOverlapFwVersion'),
+            key = cms.string(objectKey)
+        ),
```

Comment:
Why is this not just read from the GT? This kind of direct access to tags is only allowed in tests files, this is not a test file, right?

Reply:
Hi @tvami we directly fetch/update desired tags via O2O process, afaik GT is not involved in the L1 O2O process

Reply:
Hi @ggovi this is the line we are discussing now in the AlCa meeting

Reply:
@tvami thanks! Is it possible to get the tag for hlt with something like "autoCond['run2_data']" ? If this is possibile, I agree it is the way to go.

Reply:
Yes, we have that key:
https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCond.py#L35

Reply:
Ok, I would propose to post-pone to a future PR the transition to the tag resolution from GT. We may need a dedicate python function for this scope.